magpie/ui/network/templates/authorize.mako

```diff
@@ -0,0 +1,32 @@
+<%inherit file="magpie.ui.home:templates/template.mako"/>
+
+<form id="authorize_node_link"
+      action="${authorize_uri}"
+      class="new-item-form"
+      method="get">
+    <table class="authorization-table">
+        <tr>
+            <td>
+                Magpie is requesting permission to link your account with the ${requesting_user_name} user's account
+                on the Magpie instance named "${node_name}".
```

Comment:
It's a good idea. But we don't actually have the canonical URL for the nodes stored anywhere.

It's not needed since we have the various endpoints stored instead. Since this is just for display purposes, I could just use the referrer URL, which is the URL the user should expect to see anyway (since that was where they would have initiated the link request from).